frontend/src/components/pomodoroview/PomodoroView.jsx

```diff
@@ -89,26 +94,36 @@ function RemainingTimeContextProvider(props) {
 }
 
 function ClockController() {
-  const { startTime, setStartTime, setClockKey, setIsClockPlaying, isUserBreaking, setIsUserBreaking } = useContext(
-    remainingTimeContext
-  );
+  const {
+    startTime,
+    setStartTime,
+    setClockKey,
+    setIsClockPlaying,
+    isUserBreaking,
+    setIsUserBreaking,
+    isClockPlaying
+  } = useContext(remainingTimeContext);
 
-  const handleGo = () => {
-    setIsClockPlaying(true);
-    setStartTime(getCurrentTime());
-  };
-  const handleHold = () => {
-    if (!isUserBreaking) {
-      submitRecord(startTime, true);
+  const [isGoing,setChangeGo] = useState('Go') // change Go/Hold button
```

Comment:
change variable name to be more clear: 
[ButtonName, setButtonName] = useState('Go') 

p.s. isxxxx is a boolean

But wondering whether this variable is actually needed, see following suggestions